docs/languages/javascript.md

```diff
@@ -219,13 +223,18 @@ Organize imports can also be done automatically when you save a JavaScript file
 The `editor.codeActionsOnSave` setting lets you configure a set of Code Actions that are run when a file is saved. For example, you can enable organize imports on save by setting:
 
 ```json
```

Comment:
Check if there's jsonc used elsewhere for the site. It might render the comments properly.

Reply:
fixed ! was because it was updated elsewhere i believe

Reply:
If it's updated elsewhere, that sounds like it was an intentional change.
It looks like it's still json after the merge conflict fix, but we can leave it as-is.